src/V3Width.cpp

```diff
@@ -1789,7 +1789,10 @@ class WidthVisitor : public AstNVisitor {
                 v3Global.rootp()->typeTablep()->addTypesp(newp);
             }
         } else if (nodep->isIO()
-                   && !(VN_IS(nodep->dtypeSkipRefp(), BasicDType)
+                   && !((VN_IS(nodep->dtypeSkipRefp(), ClassRefDType)
+                         && VN_CAST(nodep->dtypeSkipRefp(), ClassRefDType)->classp()->name()
+                                == "process")
```

Comment:
The user could have a "process" class in a module which overrides the std::package class.  So please add a bool in AstClass that indicates this is the special process class.

Reply:
Right, this part was temporary. I was actually wondering why returning class instances is unsupported, because it actually seems to work correctly. Are there some specific cases where it breaks?

Reply:
Probably works, just need some tests to prove it out and especially that the object reference count for auto-deletion works correctly (i.e. doesn't leak or have dangling pointers).